lib/dolly/document.rb

```diff
@@ -51,7 +51,8 @@ def rev= value
       doc['_rev'] = value
     end
 
-    def save
+    def save options = {}
+      return false unless options[:validate] == false || valid?
```

Comment:
Definitely, dolly should work independently of rails but implementing validations and rails independence aren't mutually exclusive imo. Regardless, I wasn't trying to suggest implementing validations rather offering a use case for making our interfaces and methods such as `save` consistent with what a rails developer would expect.